index.html

```diff
@@ -559,23 +569,36 @@ <h3>
           steps:
         </p>
         <ol class="algorithm">
+          <li>Let |options| be the first argument.
+          </li>
+          <li>Let |signal| be the |options| dictionary member
+            of the same name if <a>present</a>, or `null` otherwise.
```

Comment:
Please default it to null in the IDL. Don’t default it here. Once the IDL is defaulted to null, you don’t need this variable. Just check option.signal.